### PR TITLE
run: `gofumpt -w -extra .`

### PR DESCRIPTION
**Pull request by Dentrax**
Enable gofumpt in golangci

Signed-off-by: Furkan <furkan.turkal@trendyol.com>

### CODE REVIEW

**Comment by Dentrax** [comment]
> lgtm, but could we add it to the makefile so people can easily update when their PRs fail?

Make sense! Added `make fmt` and updated the doc.

File: Makefile [hunk 1]
```diff
@@ -137,6 +140,12 @@ ifndef HAS_GOLANGCI
 endif
 	./_output/bin/golangci-lint run
 
+fmt:
+ifndef HAS_GOFUMPT
```
Comment (a7i) [inline Makefile:144]: thoughts on adding it to the script used by `verify-gofmt`?

Comment (Dentrax) [reply Makefile:144]: actually golangci-lint already verifies it. should we also add a secondary check anyway?

Comment (a7i) [reply Makefile:144]: I don't think that's necessary. @damemi ?

Comment (damemi) [reply Makefile:144]: It looks like `make verify` already runs `lint`, so we don't need a second check then. As long as there is a local way to verify that this has been updated

Comment (Dentrax) [reply Makefile:144]: please also notice that this `make fmt` is to run gofumpt formatter on every `**/*.go` file. (excludes ./vendor by default)

Comment (a7i) [reply Makefile:144]: @damemi are you suggesting that we should remove `make fmt` since `lint` already runs this?

**Comment by a7i** [comment]
@Dentrax sorry for the delay on this PR. would you please rebase so we can move forward on the review?

**Comment by Dentrax** [comment]
Rebased.

**Review by damemi — COMMENTED** [review]
/approve

**Pull request merged**